### PR TITLE
Move react config to separate package

### LINKED ISSUE #90: Move react preset to separate package
See https://github.com/ymaps/codestyle/pull/89#issuecomment-311020107

### PR DESCRIPTION
**Pull request by ikokostya**
Fixes #90

/cc @sigorilla

### CODE REVIEW

File: packages/eslint-config-loris-react/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# eslint-config-loris-react
+
+ESLint config for [React JSX](https://facebook.github.io/react/docs/jsx-in-depth.html).
+
+## Installation
+
+First install [eslint-config-loris](../eslint-config-loris) package. Then
```
Comment (sigorilla) [inline packages/eslint-config-loris-react/README.md]: `eslint-config-loris` is not required, so it's possible not to mention here

**Comment by ikokostya** [comment]
Done

**Comment by ikokostya** [comment]
Publish `+ eslint-config-loris-react@1.0.0`. Also add you to owners.

**Comment by sigorilla** [comment]
Thank you!

**Pull request merged**